manifests/06_config.yaml

```diff
@@ -1,10 +0,0 @@
-# TODO: axe this file in favor of EnsureOperatorConfigExists within the operator
-apiVersion: operator.openshift.io/v1
-kind: ServiceCA
-metadata:
-  name: cluster
-spec:
-  managementState: Managed
-  version: 4.0.0
-  logLevel: "4"
-  replicas: 1
```

Comment:
nit: confusing commit message. This one is a duplicate. The other one existing has `release.openshift.io/create-only: "true"` set.

Reply:
Done